recipes-extended/logrotate/logrotate_3.%.bbappend

```diff
@@ -3,12 +3,16 @@ FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
 SRC_URI += " \
 		file://logrotate.crontab \
 		file://logrotate.conf \
+		file://btmp \
+		file://wtmp \
 "
 
 do_install_append(){
     rm -rf ${D}${sysconfdir}/cron.daily/logrotate
     install -d ${D}${sysconfdir}/cron.d
     cat ${WORKDIR}/logrotate.crontab >> ${D}${sysconfdir}/cron.d/logrotate
     install -p -m 644 ${WORKDIR}/logrotate.conf ${D}${sysconfdir}/logrotate.conf
+    install -p -m 644 ${WORKDIR}/btmp ${D}${sysconfdir}/logrotate.d/btmp
+    install -p -m 644 ${WORKDIR}/wtmp ${D}${sysconfdir}/logrotate.d/wtmp
```

Comment:
Could you include a comment above this `do_install_append()` which explains that: the `btmp` and `wtmp` runparts conf files are owned by `logrotate`, that upstream's recipe installs example versions of the file, and that we're intentionally overwriting them with our own versions here. It's a nuanced maneuver that I don't think will be obvious to a future dev based on the commit message alone.

Reply:
Good idea, it was not obvious to me when I first looked at this.